dev/io.openliberty.microprofile.opentracing.2.0.internal_fat_tck/fat/src/io/openliberty/microprofile/opentracing/internal/tck/FATSuite.java

```diff
@@ -30,7 +30,7 @@
     OpentracingTCKLauncher.class,
 // Temporary comment this out until MicroProfile-4.0 is available ... including MP Rest Client 2.0
 //    OpentracingTCKLauncherMicroProfile.class,
```

Comment:
Should be able to re-enable this as well once https://github.com/OpenLiberty/open-liberty/pull/14560 is merged